tools/dds/realdds-py/python.cpp

```diff
@@ -372,28 +372,26 @@ PYBIND11_MODULE(NAME, m) {
         .def( "num_of_sensors", &dds_device::num_of_sensors )
         .def( FN_FWD( dds_device,
             foreach_sensor,
-            ( size_t, std::string const & ),
-            ( size_t sensor_index, std::string const & name ),
-            callback( sensor_index, name ); ) )
+            ( std::string const & ),
+            ( std::string const & name ),
+            callback( name ); ) )
         .def( "foreach_video_profile",
               []( dds_device const & self,
-                  size_t sensor_index,
+                  std::string group_name,
```

Comment:
Are we sure we don't want the group name to be known at the client side?
Even if we only log a warning if an unknown group name received?

Reply:
Group names are received in the client side, and I added an error when trying to call an unknown group

Reply:
👍